docs/en/latest/router-radixtree.md

```diff
@@ -218,6 +218,31 @@ $ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f
 
 This route will require the request header `host` equal `iresty.com`, request cookie key `_device_id` equal `a66f0cdc4ba2df8c096f74c9110163a9` etc.
 
+### How to filter route by POST form attributes {#post_args}
```

Comment:
We don't need `{#post_args}` now. The anchor is not ambiguous.

Reply:
If the value of the anchor is not specified, a mismatch will occur between the English document and the Chinese document, is this really not necessary?

Reply:
Just using the name displayed in the title is enough. It's quite strange to use a custom anchor (and hard to remember too).

Reply:
OK, I've removed it.